docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,40 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier, and faster!
+
+### How to use template repositories for an assignment?
+
+
+![Searches for repository, then selects the option of Importing starter code with template repositories](https://user-images.githubusercontent.com/1490434/61005360-10631080-a336-11e9-843b-ddb3e4f011ca.gif)
+
+
+When you pick your template repository as a starter code repository, you will now get two options:
+
+  * Import starter code using a template repository.
+  * Import starter code using source importer.
+  * After selecting either of the options, the following steps are the same as before. More information on this can be found on our [Videos page](/videos)
+
+Note: If the picked starter code repository is not a template repository we will try to make it into one for you. If that fails, you can do it manually from the settings page of the repository on GitHub. More details on this page: [https://help.github.com/en/articles/creating-a-template-repository](https://help.github.com/en/articles/creating-a-template-repository)
+
+
+### What are the advantages of using template repositories?
+
+* Importing starter code is faster, as it uses the newer and faster [template repositories API](https://developer.github.com/v3/repos/#create-repository-using-a-repository-template).
+* Template repositories can be really useful if only the lastest files in the starter code repository are important and you do not want the entire commit history.
```

Comment:
This seems more like a disadvantage and might be best to leave it in the "what do you miss out" section